torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -627,6 +638,9 @@ void PyTorchModelLoader::populateNodeLoaderMapping() {
   nodeLoaderMapping_[at::Symbol::fromQualString("aten::exp")] =
       [this](const torch::jit::Node *node) { return loadExp(node); };
 
+  nodeLoaderMapping_[at::Symbol::fromQualString("aten::sqrt")] =
```

Comment:
there's also sqrt_

Reply:
What's the difference? How do I get that to show up in the PT IR instead of `sqrt`?

Reply:
sqrt_ is inplace (all foo_ are in place version of foo). You can just call b.sqrt_().